kapitan/utils.py

```diff
@@ -624,3 +629,14 @@ def safe_copy_tree(src, dst):
                 outputs.append(dst_name)
 
     return outputs
+
+
+def copy_tree(src, dst):
+    """Recursively copy a given directory from `src` to `dst`.
+
+    Returns a list of the copied files.
+    """
+    before = set(glob.iglob("*", recursive=True))
```

Comment:
Shouldn't that be `glob.iglob(f"{dst}/**", recursive=True))` instead?

Reply:
same for "after" I guess

Reply:
I can't test it within kapitan. I just found this function while looking for a replacement for distutils `copy_tree` that also returns the list of files copied. I'm not a kapitan user and don't even know what that is :)

Reply:
Fair :) I'll fix it, thank you for reporting it, much appreciated!